nbextensions/bulkImportCell/cellControlPanel.js

```diff
@@ -13,82 +13,80 @@ define([
-         *       {
-         *          configure: {
-         *              label: "Configure"
-         *          }
-         *          , etc.
-         *       }
-         * - actions: object that defines the action buttons:
-         *   - current: which is the current button, and its state:
-         *     - name - string, one of the action keys
-         *     - disabled - if truthy, then should not be clickable
-         *   - availableButtons - the set of available buttons, each key
-         *      is the button name, values have the following properties:
-         *      - help - tooltip string
-         *      - type - Bootstrap class types
-         *      - classes - array of classes to add to the button component
-         *      - label - button text
-         *   - runAction - function that gets run when the button is clicked,
-         *      takes the button name as the single parameter
-         * @param {object} options
-         */
-        constructor(options) {
-            this.bus = options.bus;
-            this.ui = options.ui;
+    /**
+     * options -
+     * - bus - the message bus
+     * - ui - the UI controller
+     * - tabs: an object with the following properties:
+     *   - toggleAction - a function that should be run when toggling tabs, takes
+     *      the tab name as a single parameter
+     *   - tabs: also an object:
+     *     - selectedTab - string, one of the keys under "tabs"
+     *     - tabs - an object where each key is a tab key, and has a display label:
+     *       {
+     *          configure: {
+     *              label: "Configure"
+     *          }
+     *          , etc.
+     *       }
+     * - actions: object that defines the action buttons:
+     *   - current: which is the current button, and its state:
+     *     - name - string, one of the action keys
+     *     - disabled - if truthy, then should not be clickable
+     *   - availableButtons - the set of available buttons, each key
+     *      is the button name, values have the following properties:
+     *      - help - tooltip string
+     *      - type - Bootstrap class types
+     *      - classes - array of classes to add to the button component
+     *      - label - button text
+     *   - runAction - function that gets run when the button is clicked,
+     *      takes the button name as the single parameter
+     * @param {object} options
+     */
+    function CellControlPanel(options) {
+        let bus = options.bus;
+        let ui = options.ui;
 
-            this.cellTabs = new CellTabs({
-                ui: this.ui,
-                bus: this.bus,
-                toggleAction: options.tabs.toggleAction,
-                tabs: options.tabs.tabs
-            });
+        // this.cellTabs = new CellTabs({
+        //     ui: this.ui,
+        //     bus: this.bus,
+        //     toggleAction: options.tabs.toggleAction,
+        //     tabs: options.tabs.tabs
+        // });
```

Comment:
Does this still need to be here if it's called from `bulkImportCell.js`?

Reply:
Nope, removed all references to tabs from the control panel.